### PR TITLE
Update packages

### LINKED ISSUE #498: update docker lighthouse to 1.1.1

### LINKED ISSUE #499: update lodestar to 0.16.0

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #498 
closes #499

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -169,12 +169,13 @@
...
-    "@chainsafe/lodestar-types": "0.13.0",
-    "@chainsafe/lodestar-utils": "0.13.0",
-    "@chainsafe/lodestar-validator": "0.13.0",
+    "@chainsafe/lodestar-beacon-state-transition": "0.16.0",
+    "@chainsafe/lodestar-config": "0.16.0",
+    "@chainsafe/lodestar-db": "0.16.0",
+    "@chainsafe/lodestar-types": "0.16.0",
+    "@chainsafe/lodestar-utils": "0.16.0",
+    "@chainsafe/lodestar-validator": "0.16.0",
+    "@chainsafe/persistent-ts": "0.16.0",
```
Comment (mpetrunic) [inline package.json:178]: do we use that directly somewhere or some package is missing dependency delcaration?

**Review by mpetrunic — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (BeroBurny) [reply package.json:178]: one of the packages use `@chainsafe/persistent-ts`

Comment (mpetrunic) [reply package.json:178]: but it will be included automatically no?

Comment (BeroBurny) [reply package.json:178]: probably package that uses it doesn't have it as a dependency

Comment (mpetrunic) [reply package.json:178]: do you have error so I can figure out where is it missing, so we can open issue. Don't push issues under the rug 😄

Comment (BeroBurny) [reply package.json:178]: done https://github.com/ChainSafe/lodestar/issues/2052

**Pull request merged**